examples/l2_rpc_client.rs

```diff
@@ -33,47 +34,48 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
     // initiate logging
     UPTransportZenoh::try_init_log_from_env();
 
-    println!("uProtocol RPC client example");
-    let zenoh_transport = UPTransportZenoh::builder("//rpc_client/1/1/0")
-        .expect("invalid URI")
+    info!("uProtocol RPC client example");
```

Comment:
I think it's fine to use `println!` in the example. We want to show the messages to users.

Reply:
Yeah, you're probably right. I'll change that back in the examples.